.github/workflows/lighthouse.yml

```diff
@@ -1,29 +1,75 @@
-name: "Run Lighthouse audit"
+name: "Run Lighthouse Audit"
 on:
-  schedule:
-  - cron: "0 0 * * *"
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - "**"
 jobs:
-  validate-508:
+  lhci:
+    name: Lighthouse
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
 
-    - name: Lighthouse CI Action
-      uses: treosh/lighthouse-ci-action@v7
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
It looks like we went back a checkout version, is that intentional?

Reply:
We are somewhat inconsistent across github actions with using v2 and v3. The difference seems to be that v3 uses node 16 whereas v2 uses node 12. It's not obvious to me which is best since we use node 14 in general. I can test it out with v3 though.

Reply:
If you can test it with v3 that would be appreciated! Otherwise I think we'll end up with a Dependabot PR to upgrade to v3 anyways 😄